src/pools/D3MAavePool.sol

```diff
@@ -86,7 +93,8 @@ contract D3MAavePool is ID3MPool {
     event File(bytes32 indexed what, address data);
     event Collect(address indexed king, address indexed gift, uint256 amt);
 
-    constructor(address hub_, address dai_, address pool_) {
+    constructor(bytes32 ilk_, address hub_, address dai_, address pool_) {
+        ilk = ilk_;
```

Comment:
rather than passing this in for each pool in the constructor (kind of hard to enforce in our current interface pattern), how would you feel about adding it to the `exit` args?

Reply:
I considered this option but eventually felt it was more an implementation detail for these two pools, that's why ended up passing the `ilk` in the constructor. If we consider this is actually something to generalize, then yeah we could move to the function param.